apps/server/swagger/data-api.yml

```diff
@@ -31,6 +31,13 @@
         schema:
           type: integer
         description: Optional query parameter to specify the number of results per page. Default value is 20
+      - name: view
+        in: query
```

Comment:
adding `view` query param on GET data endpoints

Reply:
This endpoint and others that fetch data as list or compound implements pagination. with default of 20 items per page. This will alleviate fetching large mount of data.

It's correct, the property `defaultCentricEntity`  is used to determine thee root schema in `compound` view